connect-examples/v2/php_checkout/templates/example-checkout-nosdk.php

```diff
@@ -0,0 +1,59 @@
+<?php
+// Include the Square Connect API resources, store config, and helper functions
+require_once '/local/path/to/sdk/SquareConnect/autoload.php';
```

Comment:
Yes. This (and the helper file) are linkd in the setup guide so we're not totally ignoring ppl not using the SDK